README.md

```diff
@@ -38,7 +38,7 @@ pip install -i https://test.pypi.org/simple/ pyimager
 ## Functions:
 
 - `circropper`: This function crops the input image into a circle. This can be useful when you want to make icons from images. 
-- `redusize`: This function reduces the dimension of a given image by removing vertical and horizontal seams.
+- `redusize`: This function reduces the dimension of a given image by removing first columns and/or rows of the image according to the desired new width/height, as in the example below.
```

Comment:
looks good but seems the order of dependencies and functions are changed in the master branch, please fix order conflicts